auth/token-managers/jwt-token-manager.ts

```diff
@@ -90,13 +90,19 @@ export class JwtTokenManager {
    *   has expired.
    */
   public getToken(): Promise<any> {
-    if (!this.tokenInfo[this.tokenName] || this.isTokenExpired()) {
+    if (!this.tokenInfo[this.tokenName]) {
```

Comment:
I agree with @dpopp07 - I think the expiry check needs to remain here so that a synchronous request for a new token can be made in the case that a token's validity has lapsed after a period with no requests to force the preemptive refresh.
I guess you could add a test for this with a very short expiry time and a short block in a test case.